jmclient/jmclient/configure.py

```diff
@@ -574,10 +560,20 @@ def get_blockchain_interface_instance(_config):
             rpc_wallet_file)
         if source == 'bitcoin-rpc': #pragma: no cover
             bc_interface = BitcoinCoreInterface(rpc, network)
+            if testnet:
+                btc.select_chain_params("bitcoin/testnet")
+            else:
+                btc.select_chain_params("bitcoin")
         elif source == 'regtest':
             bc_interface = RegtestBitcoinCoreInterface(rpc)
+            btc.select_chain_params("bitcoin/regtest")
         elif source == "bitcoin-rpc-no-history":
             bc_interface = BitcoinCoreNoHistoryInterface(rpc, network)
+            if testnet or network == "regtest":
+                # TODO will not work for bech32 regtest addresses:
```

Comment:
I'm surprised this doesn't make the tests fail

Reply:
Very good point. I will check it out.